### PR TITLE
[DOCS] New artifact card format in Manage pages

### PR DESCRIPTION
**Pull request by joepeeples**
Addresses #1232. The format and style of artifact cards has changed in these parts of the Security app, resulting in slightly different procedures for editing and deleting each entry. This PR revises those procedures, and also aligns the structure of the procedures to be consistent between the pages. No screenshot updates needed in this PR, because they were already updated in other PRs.

Previews:
- [Trusted applications](https://security-docs_1235.docs-preview.app.elstc.co/guide/en/security/master/trusted-apps-ov.html#trusted-apps-list)
- [Event filters](https://security-docs_1235.docs-preview.app.elstc.co/guide/en/security/master/event-filters.html#manage-event-filters)

### CODE REVIEW

**Review by benironside — COMMENTED** [review]
LGTM, added two inline comments. 

More generally, I wonder if it would be helpful to make these into H3 or H4:
"To edit a trusted application:", "To delete a trusted application:", "To edit an event filter", "To delete an event filter". Someone might want to skip to just one of those sections and to my eye, they blend in. 

Also on the preview of the event filters page, step 2 needs some extra whitespace above it.

File: docs/management/admin/event-filters.asciidoc [hunk 1]
```diff
@@ -33,24 +33,20 @@ image::images/event-filter.png[]
...
-* `comments`
-* `value`
+To refine the Event filters list by a specific value, enter a query in the search bar. You can search the by name, comments, or the value of a field.
 
 To edit an event filter:
 
-. Click *Edit* on the appropriate event filter card.
-. Modify details or conditions as appropriate.
-. Click *Update event filter*. The newly modified filter appears at the top.
+. Click the actions button (…​) for the event filter you want to edit, then select *Edit event filter*.
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/management/admin/event-filters.asciidoc:45]: Yes, good catch!

**Comment by joepeeples** [comment]
> More generally, I wonder if it would be helpful to make these into H3 or H4: "To edit a trusted application:", "To delete a trusted application:", "To edit an event filter", "To delete an event filter". Someone might want to skip to just one of those sections and to my eye, they blend in.
> 
> Also on the preview of the event filters page, step 2 needs some extra whitespace above it.

@benironside I went back and forth about including headings -- the procedures are so short I thought headings might feel overly constructed. But if lack of headings seemed noticeable to you, that's significant too. @jmikell821 @nastasha-solomon: Thoughts on headings/no headings?

For the whitespace, we don't have much control over design details like this with Asciidoc, though I agree the line spacing is a little uneven.

File: docs/management/admin/event-filters.asciidoc (hunk 1 above)
Comment (joepeeples) [inline docs/management/admin/event-filters.asciidoc:45]:
```suggestion
. Click the actions button (*…​*) for the event filter you want to edit, then select *Edit event filter*.
```

File: docs/management/admin/event-filters.asciidoc [hunk 2]
```diff
@@ -33,24 +33,20 @@ image::images/event-filter.png[]
...
 
 [role="screenshot"]
 image::images/event-filters-list.png[]
 
-To refine the Event filters list by a specific value, enter a query in the https://elastic.github.io/eui/#/forms/search-bar[search bar]. You can search the following attributes:
-
-* `name`
-* `comments`
-* `value`
+To refine the Event filters list by a specific value, enter a query in the search bar. You can search the by name, comments, or the value of a field.
```
Comment (joepeeples) [inline docs/management/admin/event-filters.asciidoc:41]:
```suggestion
To refine the Event filters list, enter a query in the search bar. You can search the by name, comments, or the value of a field.
```

File: docs/management/admin/trusted-apps.asciidoc [hunk 3]
```diff
@@ -33,33 +33,22 @@ NOTE: You can only add a single field type value per trusted application. For ex
...
 [[trusted-apps-list]]
-== Trusted applications list
+== View and manage trusted applications
 
 The *Trusted applications* list displays all the trusted applications that have been added to the {security-app}.
 
 [role="screenshot"]
 image::images/trusted-apps-list.png[]
 
+To refine the Trusted applications list by a specific value, enter a query in the search bar. You can search by name, description, or the value of a field.
```
Comment (joepeeples) [inline docs/management/admin/trusted-apps.asciidoc:43]:
```suggestion
To refine the Trusted applications list, enter a query in the search bar. You can search by name, description, or the value of a field.
```

**Comment by nastasha-solomon** [comment]
@joepeeples hm, I can see both sides. On one hand, this is already a short section, and adding headers would break it down even more and would be adding duplicate content. On the other hand, headers would create section links in the mini TOC in the top right corner of each topic (see image below), which might improve the topic's scanability and the user's navigation experience. 


headers

Headers would also likely improve user experience if the user was consuming this topic via a screen reader. In fact, accessibility guidelines generally advocate for headers on web pages because of the structure and organization they offer. 

Personally, I think both approaches work. If you chose to go the non-header route, I would recommend placing the sentences under the screenshot ("To refine the Event filters/Trusted applications list, enter a query in the search bar. You can search them by name, comments, or the value of a field.") before the screenshot. Then there'd be nothing but procedures under the image and it'd be uber obvious that those were steps to managing the event filters and trusted apps.

**Comment by joepeeples** [comment]
I think I'm convinced that headers would be good, esp for accessibility and usability. I also like the idea of moving the "To refine the X list" above the image. Stay tuned for revisions soon...

**Comment by joepeeples** [comment]
OK @nastasha-solomon @benironside, I added headings for those sections.

**Review by nastasha-solomon — APPROVED** [review]
LGTM!

**Review by benironside — APPROVED** [review]

File: docs/management/admin/event-filters.asciidoc [hunk 4]
```diff
@@ -33,24 +33,26 @@ image::images/event-filter.png[]
...
-
+[discrete]
+[[edit-event-filter]]
+=== Edit an event filter
 To edit an event filter:
 
-. Click *Edit* on the appropriate event filter card.
-. Modify details or conditions as appropriate.
-. Click *Update event filter*. The newly modified filter appears at the top.
+. Click the actions button (*...*) for the event filter you want to edit, then select *Edit event filter*.
```
Comment (jmikell821) [inline docs/management/admin/event-filters.asciidoc:48]: Should we cap the button title, i.e., `Actions button` even if there isn't a UI label?

**Review by jmikell821 — COMMENTED** [review]
Just had one question/comment.

File: docs/management/admin/event-filters.asciidoc (hunk 4 above)
Comment (joepeeples) [reply docs/management/admin/event-filters.asciidoc:48]: I'd prefer lowercase if there isn't a UI label or any other specific reason to capitalize.

**Comment by muskangulati-qasource** [comment]
Hi @jmikell821,

We tested the linked docs and found that all the docs are correctly updated as per the latest UI. So we are good to go ahead and merge the changes.

Marking this ticket as 'QA validated'.

Thanks!

**Pull request merged**